library/Fission/User/DID.hs

```diff
@@ -1,6 +1,13 @@
 module Fission.User.DID
-  ( -- * Reexports
+  ( txtFromKey
+  , -- * Reexports
     module Fission.User.DID.Types
   ) where
 
+import Fission.Prelude
 import Fission.User.DID.Types
+
+import qualified Fission.Key as Key
+
+txtFromKey :: Key.Public -> Text
+txtFromKey pk = textDisplay (DID pk Key)
```

Comment:
The convention would be to call this `toText`.

That said, what we actually want here is `textDisplay`:

```haskell
instance Display Key.Public where
  textDisplay pk = textDisplay (DID pk Key)
```

Reply:
This actually takes a key, and returns the DID txt, which is not the same as just displaying the key

when I call it, I call `DID.txtFromKey`. maybe the clearer name would be `Key.toDIDText`?

Reply:
How about two functions, and compose them at the call site? (more idiomatic)

```haskell
textDisplay $ toDID pk
```

Reply:
A few reasons why this is idiomaic:

* Composition is considered par for the course
* Prevent a proliferation of `xToY` functions (grows factorially)
* Still reads well (“tells a story”)

Reply:
:+1: 
I actually switched the order of the `DID` type from `DID pk Key` to `DID Key pk` to make it easier to compose, and we don't even need a `toDID` fn :ok_hand:

Reply:
Amazing 👍